resources/views/cart/partials/product/price.blade.php

```diff
@@ -1,14 +1,14 @@
 <td class="flex items-center font-medium max-md:w-1/3 md:table-cell">
-    <div v-if="item.specialPrice">
-        @{{ item.specialPrice | price }}
+    <div v-if="item.prices.specialPrice">
+        @{{ item.prices.specialPrice | price }}
     </div>
-    <div :class="{ 'line-through text-xs text-ct-inactive font-normal': item.specialPrice }">
-        @{{ item.price | price }}
+    <div :class="{ 'line-through text-xs text-ct-inactive font-normal': item.prices.specialPrice }">
+        @{{ item.prices.price_including_tax.value | price }}
     </div>
 </td>
 <td class="flex items-center font-medium max-md:w-1/3 md:table-cell [&>*]:mx-auto">
```

Comment:
you can use *:mx-auto

Reply:
Wouldn't that select the current element as well as containing elements, while this selector only selects all child elements?